package-overrides/npm/bootstrap@3.3.7.json

```diff
@@ -0,0 +1,15 @@
+{
+  "main": "js/bootstrap",
+  "directories": {
+    "lib": "dist"
```

Comment:
This is a breaking change, do we really need this. It can be better to use `dist/js/bootstrap` etc in the other configurations.